### PR TITLE
login and registration pages restyled

### PR DESCRIPTION
**Pull request by dragosdinescu**

### CODE REVIEW

File: src/main/webapp/WEB-INF/pages/register.jsp [hunk 1]
```diff
@@ -1,108 +1,105 @@
...
+                        <div class="invalid-feedback">
+                            Please confirm the password
+                        </div>
+                    </div>
+                    <div class="field">
+                        <input type="submit" value="Register">
+                    </div>
+                    <div class="signup-link">
+                        Already a member? <a href="/WEB-INF/pages/login.jsp"">Log in</a> <%-- To do for later (if time allows) --%>
+                    </div>
```
Comment (rmoldo) [inline src/main/webapp/WEB-INF/pages/register.jsp]: Can you comment this whole section

File: src/main/webapp/WEB-INF/pages/login.jsp [hunk 2]
```diff
@@ -6,20 +6,42 @@
...
+                    <div class="field">
+                        <input type="password" id="password" name="j_password" required>
+                        <label>Password</label>
+                    </div>
+                    <div class="field">
+                        <input type="submit" value="Login">
+                    </div>
+                    <div class="signup-link"> 
+                        Not a member? <a href="login.jsp">Sign up now</a> <%-- To do for later (if time allows) --%>
+                    </div>
```
Comment (rmoldo) [inline src/main/webapp/WEB-INF/pages/login.jsp]: Can you comment this whole section?

**Review by ghost — APPROVED** [review]

File: src/main/webapp/WEB-INF/pages/register.jsp [hunk 3]
```diff
@@ -1,108 +1,105 @@
 <%-- 
-    Document   : register
-    Created on : Dec 28, 2020, 2:40:44 PM
+    Document   : login
```
Comment (ghost) [inline src/main/webapp/WEB-INF/pages/register.jsp:2]: Maybe this shouldn't be changed, but nah

**Review by rmoldo — APPROVED** [review]

File: src/main/webapp/WEB-INF/pages/register.jsp (hunk 1 above)
Comment (dragosdinescu) [reply src/main/webapp/WEB-INF/pages/register.jsp]: Done, commented html + css section

File: src/main/webapp/WEB-INF/pages/login.jsp (hunk 2 above)
Comment (dragosdinescu) [reply src/main/webapp/WEB-INF/pages/login.jsp]: Done, commented html + css section

**Pull request merged**